src/android/build-extras.gradle

```diff
@@ -0,0 +1,50 @@
+// Needed to speed up asset listing: refer to https://github.com/apache/cordova-plugin-file/blob/76c8fd5e43598ea660c2db64eb08e2659b45a575/src/android/AssetFilesystem.java#L41
```

Comment:
Asset listing did not show a significant difference, but recursive copying of the www contents in a blank Cordova app with CodePush added took 4.26 seconds with the index and 7.83 seconds without the index.